examples/multiphysics/static-bubble-chns/static-bubble-multiple-folders.py

```diff
@@ -0,0 +1,119 @@
+"""
+Quick user guide: 
+
+This python script is used to plot a slice of the pressure field and the difference between the averaged pressure outside of the bubble and inside the bubble.
+
+How to use this script?
+
+1- Create a first directory (we will call it outputs but feel free to name it as you like).
+2- For each simulation, create a new directory in outputs : output1, output2, output3,...
+3- Run the simulations and store the results in their correct directories
+4- Execute the script with the path of outputs as argument :
+   python3 multiple_folders_new_bubble_detachment_post_processing /PATH/TO/OUTPUTS/DIR 
+5- Enjoy your plots. 
+
+This should be called as follows : python3 static-bubble-multiple-folders.py /PATH/TO/OUTPUTS/DIR
+"""
+# -------------------------------------------
+# Modules
+# -------------------------------------------
+
+from postprocessing_static_bubble import get_pressure_difference, analytical_solution,get_pressure_slice
+import numpy as np
+import os
+import sys
+import scienceplots
+from natsort import os_sorted
+import matplotlib.pyplot as plt
+
+# For plotting nice Latex-style plots and controlling the sizes of the texts on
+# the plot
+plt.style.use(['science', 'ieee'])
+
+SMALL_SIZE = 10
+MEDIUM_SIZE = 10
+BIGGER_SIZE = 15
+plt.rc('font', size=SMALL_SIZE)  # controls default text sizes
+plt.rc('axes', titlesize=SMALL_SIZE)  # fontsize of the axes title
+plt.rc('axes', labelsize=8)  # fontsize of the x and y labels
```

Comment:
Is the label size supposed to be hardcoded here?

Reply:
I prefer hard coding all the legend parameters, although tedious, the figures are much cleaner this way